docs/eventing/samples/writing-event-source/README.md

```diff
@@ -11,10 +11,7 @@ well as receive adapter, which events can be viewed through a basic
 Just want to see the code? The reference project is
 [https://github.com/knative-sandbox/sample-source](https://github.com/knative-sandbox/sample-source).
 
-A variety of event sources are available in Knative [`eventing-contrib`](https://github.com/knative/eventing-contrib/) such as
-[`KafkaSource`](https://github.com/knative/eventing-contrib/tree/master/kafka/source),
-[`GithubSource`](https://github.com/knative/eventing-contrib/tree/master/github) and
-[`AWSSQSSource`](https://github.com/knative/eventing-contrib/tree/master/awssqs) that can be used as a reference.
+[Knative Sources](../../sources/_index.md#knative-sources) can be used as a reference.
```

Comment:
```suggestion
[Knative Sources](../../sources/#knative-sources) can be used as a reference.
```

All `_file.ext` are internal build files and should not be directly linked to. This should fix this (note that this is not the convention the rest of the library uses). See comment below for more context.

Reply:
yeah, I was actually wondering about that! 
Thanks for the suggestion